dace/runtime/include/dace/cuda/dynmap.cuh

```diff
@@ -11,6 +11,11 @@
 #include <hip/hip_cooperative_groups.h>
 #endif*/
 
+// HIP does not yet support features in cooperative groups used here.
+/*#elif defined(__HIPCC__)
+#include <hip/hip_cooperative_groups.h>
+#endif*/
+
```

Comment:
Something weird happened

Reply:
I don't know what had happened here. I again just copy and paste the content from the master branch.